cloudinit/net/sysconfig.py

```diff
@@ -99,13 +99,18 @@ def drop(self, key):
     def __len__(self):
         return len(self._conf)
 
+    def str_skipped(self, key, val):
```

Comment:
`str_skipped` is a slightly odd name: it's operating on two values (not **a** `str`) and, looking at the `to_string` implementation, we expect `val` could be a `bool` or any other type (not a **`str`**). `skipped` is also a strange tense: that implies to me that this is reporting on something that happened ("was this string skipped?") whereas it is actually determining what we should do ("should we skip this line?").

I'd like to see a more accurate name for it, as well as a docstring which briefly describes what it does (we've got to start somewhere!).

Reply:
I'm going with `skip_key_value()` with the docstring `Skip the pair key, value if it matches a certain rule.`. It reads well in the code, it checks if the pair should be skipped or not.